target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-rutx10.dts

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4018-rutx.dtsi"
```

Comment:
Why do we need a DTSI at all here, just for one device?

Reply:
Teltonika has several RUTX devices which share the same platform. I already separated the common parts into the DTSI. Makes adding other devices easier.